joomla/joomla-cms
# Allowing to exclude menu items from displaying in frontend menus (in menu item)

#### Description

This is PR takes the contributions from https://github.com/joomla/joomla-cms/pull/8814 and adds the ability to exclude menu items from displaying in frontend menus.

For more info see https://github.com/joomla/joomla-cms/pull/8814
#### How to test
1. Fresh Joomla install with multilanguage.
2. Apply this patch
3. Go to a menu item that is displayed in the frontend main menu, separator "Link Type" and set the option "Display in Menu Modules" to "No" to exclude the menu item from displaying in the frontend menu.
4. Check in the frontend it those items are not displayed in the main menu.
#### Observations

Suggestions are welcome.
